kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/packages/PackagesPanelUi.ui.xml

```diff
@@ -125,7 +124,7 @@
             <b:ModalBody>
                 <b:Panel>
                     <b:PanelBody>
-                        <b:Alert type="WARNING">
+                        <b:Alert type="DANGER">
```

Comment:
Please change header to: "Copyright (c) 2011, 2020 Eurotech and/or its affiliates"